lib/cn/csched_sp3.c

```diff
@@ -1287,13 +1289,17 @@ sp3_work_complete(struct csched *handle, struct cn_compaction_work *w)
     struct sp3 *sp = (struct sp3 *)handle;
 
     if (w->cw_action == CN_ACTION_SPLIT) {
-        struct sp3_node *spn = tn2spn(w->cw_split.nodev[0]);
+        if (w->cw_split.nodev[0]) {
```

Comment:
Are your LEFT & RIGHT enums visible here? Might be useful to expose them and use them here :)

Reply:
It's all used within node_split.c, kvset_split.c and cn_tree.c.. I will think about exposing them here probably in a different PR.